Misc/NEWS.d/next/Library/2024-08-27-12-11-00.gh-issue-123363.gKuJp6.rst

```diff
@@ -0,0 +1 @@
+Show string value of :opcode:`CONTAINS_OP` oparg in :mod:`dis` output.
```

Comment:
```rst
Show string value of :opcode:`CONTAINS_OP` oparg in :mod:`dis` output.
Patch by [...].
```

If you want, you can also credit yourself (just be sure the end the sentence with a period).